pyfr/solvers/baseadvec/elements.py

```diff
@@ -158,14 +158,20 @@ def modal_filter(uin):
                                               initval=entmin_int)
 
             # Setup nodal/modal operator matrices
-            self.invvdm = self._be.const_matrix(self.basis.ubasis.invvdm.T)
-            if self.basis.fpts_in_upts:
-                self.vdm_ef = self._be.const_matrix(self.basis.ubasis.vdm.T)
-                self.m0 = None
+            linearize = self.cfg.getbool('solver-entropy-filter', 'linearize',
```

Comment:
Check if the default argument is needed here.

Reply:
I think by default having it off wouldn't be a bad idea (just for user-friendliness), but I can remove it too.

Reply:
We have the `getbool('solver-entropy-filter', 'linearize', False)` twice in the PR.  Generally, we try and only use the default argument once and rely on the fact that when a default is used there is an implicit set (so future accesses do not need the default to be specified).